src/App.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setSearch] = useState('');
+
+  const visibleMovies = (moviesFromServer.filter((movie) => {
+    const { title, description } = movie;
+
+    return title.toLowerCase().includes(query.toLowerCase())
+      || description.toLowerCase().includes(query.toLowerCase());
+  }));
```

Comment:
You can create separate value for ```includes(query.toLowerCase()```. Don't repeat yourselfe

Reply:
it's not that critical, but indeed makes sense – avoid repeating yourself
proper decomposition is everything